src/ui/components/Difference/index.jsx

```diff
@@ -1,5 +1,141 @@
+import React, { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
+import styled from "styled-components";
+
+import Description from "../shared/Description";
+import Button from "../shared/Button";
+
 function Difference() {
-  return <h1>Difference</h1>;
+  const navigate = useNavigate();
+  const [displayText, setDisplayText] = useState({
+    text: "변경사항을 선택해주세요.",
+    className: "default",
+  });
+
+  const processDifferences = ({ differenceInformation }) => {
+    const NEW_NODE = "선택하신 이전 버전에는 존재하지 않는 노드입니다!";
+    const NEW_FRAME = "선택하신 이전 버전에는 존재하지 않는 프레임입니다!";
+
+    if (
+      differenceInformation === NEW_NODE ||
+      differenceInformation === NEW_FRAME
```

Comment:
- `P1`
```suggestion
const NEW_NODE = "선택하신 이전 버전에는 존재하지 않는 노드입니다!";
const NEW_FRAME = "선택하신 이전 버전에는 존재하지 않는 프레임입니다!";

functiom Difference() {
  const navigate = useNavigate();
  const [displayText, setDisplayText] = useState({
    text: "변경사항을 선택해주세요.",
    className: "default",
  });

  const processDifferences = ({ differenceInformation }) => {
    if (
      differenceInformation === NEW_NODE ||
      differenceInformation === NEW_FRAME
```

상수는 컴포넌트 외부에 작성하는게 일반적인 것 같습니다!

Reply:
상수와 함께 함수를 따로 분리해두었습니다 :)